bowtie/_cli.py

```diff
@@ -715,6 +715,21 @@ def convert(
         )
 
 
+def _set_dialect(ctx: click.Context, _, value: _Dialect):
+    """
+    Sets the dialect according to $schema if specified.
+    """
+    if value:
+        return value
+    schema = ctx.params.get("schema")
+    dialect_from_schema = schema.get("$schema") if schema else None
```

Comment:
This looks like it will blow up if `schema` is the `True` schema.

Reply:
Oh yeah you are right!